students/Sally_Shen/lesson2/series .py

```diff
@@ -0,0 +1,30 @@
+'''''Goal: The Fibonacci Series is a numeric series starting with the integers 0 and 1.
```

Comment:
careful! looks like you have a space before the dot in the filename -- that could cause problems iwth the command line.

you can rename it in git with:

git mv "series .py" series.py

you need the quotes so it will read the space as part of the filename.